mubench.reviewsite/src/Controller/DownloadController.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace MuBench\ReviewSite\Controller;
+
+use Monolog\Logger;
+use MuBench\ReviewSite\Model\DetectorResult;
+use MuBench\ReviewSite\Model\ExperimentResult;
+use MuBench\ReviewSite\Model\Misuse;
+use MuBench\ReviewSite\Model\ReviewState;
+use MuBench\ReviewSite\CSVHelper;
+use MuBench\ReviewSite\DBConnection;
+use Slim\Http\Request;
+use Slim\Http\Response;
+
+class DownloadController
+{
+    private $db;
+    private $logger;
+    private $default_ex2_review_size;
+    private $csv_helper;
+
+    public function __construct(DBConnection $db, Logger $logger, $default_ex2_review_size)
+    {
+        $this->db = $db;
+        $this->logger = $logger;
+        $this->default_ex2_review_size = $default_ex2_review_size;
+        $this->csv_helper = new CSVHelper();
+    }
+
+    public function download_stats(Request $request, Response $response, array $args)
+    {
+        $ex2_review_size = $request->getQueryParam("ex2_review_size", $this->default_ex2_review_size);
+        $experiment = $args['exp'];
+        $stats = [];
+        $detectors = $this->db->getDetectors($experiment);
+        foreach ($detectors as $detector) {
+            $runs = $this->db->getRuns($detector, $experiment, $ex2_review_size);
+            if (strcmp($experiment, "ex2") === 0) {
```

Comment:
This filtering is to ease the review workflow in the UI. We shouldn't do it for data export. Please just include everything in the CSV.